lib/cli/package.json

```diff
@@ -20,7 +20,7 @@
     "dist": "rm -rf ../../dist/libs/cli && npm run build && cp -R ./bin ../../dist/libs/cli && cp -R ./resources ../../dist/libs/cli && cp -R ./templates ../../dist/libs/cli && cp ./package.json ../../dist/libs/cli"
   },
   "dependencies": {
-    "@alfresco/js-api": ">=8.0.0-alpha.4",
+    "@alfresco/js-api": ">=8.0.0-alpha.6",
```

Comment:
Should I revert this or keep it?

Reply:
Revert it, it always has to point to previous version during the release, later on it can point to the alpha but this can be changed only after the release

Reply:
Oh and don't forget to rollback the change for it's lock file as well

Reply:
Also, does it need to be >=8.0.0-alpha.4 or just 8.0.0-alpha.4?

Reply:
Both should be correct, usually we keep it as  >=8.0.0-alpha.4

Reply:
The package-lock.json only has changes that mention the version of the cli library itself, not any changes related to what version of js-api to use

Reply:
Ah right, all good then